sonaranalyzer-dotnet/rspec/cs/S4143_c#.json

```diff
@@ -0,0 +1,16 @@
+{
+  "title": "Dictionary values should not be replaced unconditionally",
```

Comment:
This works for both dictionaries and collections/arrays, we should probably rename in RSPEC... It is weird that the original title says `Map` which imo translates to `Dictionary`, but all other languages renamed to `Collection`.

Reply:
Agreed - having "dictionary" in the title is definitely confusing.

"Values in indexed/keyed collections should not be..."?
"Values in dictionaries, arrays and collections should not be..."?